### PR TITLE
Added basic layout to homepage 

### PR DESCRIPTION
**Pull request by Komal914**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (nayeonshin) [inline .vscode/launch.json]: Why do we need this file?

File: client/src/components/FriendList.js [hunk 2]
```diff
@@ -1 +1,11 @@
 import React from "react";
+
+function FriendList() {
+    return (
+      <div className="friendList">
```
Comment (nayeonshin) [inline client/src/components/FriendList.js:5]: Can you please change all the class names to kebab case (In this case, `friendList` --> `friend-list`)? (You'll also need to change the CSS selector accordingly)

**Review by nayeonshin — CHANGES_REQUESTED** [review]

**Review by panzerama — APPROVED** [review]

File: client/src/components/FriendList.js (hunk 2 above)
Comment (nayeonshin) [inline client/src/components/FriendList.js:5]: Could you please use kebab case for all the class names?

File: .vscode/launch.json (hunk 1 above)
Comment (Komal914) [reply .vscode/launch.json]: I don't think we need it, we can delete it.

**Comment by Komal914** [comment]
Closing pull request for homepage layout till we have time to work on it.

**Pull request closed without merging**